src/components/Board.vue

```diff
@@ -1,89 +1,129 @@
 <template>
-  <div id="game">
-    <div id="score">0</div>
+  <div id="game" ref="game">
+    <div id="score">
+      {{ this.$store.state.scoreStore.score }}
+    </div>
     <button class="button-start" @click="startGame">start</button>
-    <div id="touch-area">
-      <div id="good">
-        <div id="excellent">
-        </div>
+    <button class="button-start" style="margin-top: 80px" @click="stopGame">stop</button>
+    <div id="good" ref="good">
+      <div id="excellent" ref="excellent">
       </div>
     </div>
+    <arrow
+      v-for="n in storeItems"
+      :key="n.id"
+    >
+    </arrow>
   </div>
 </template>
 
 <script lang="ts">
 
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
-import { Block } from '@/types'
-
-@Component
+import Arrow from '@/gameComponents/Arrow.vue'
+import { EDirection, IFlowProps } from '@/types'
+import { StoreModuleEnum } from '@/store/types'
+import { EActionScore } from '@/store/modules/score/typesScore'
+import { EActionArrow, IArrowData } from '@/store/modules/arrow/typesArrow'
+
+@Component({
+  components: {
+    Arrow
+  }
+})
 export default class Board extends Vue {
-  gameFlow = document.getElementById('game')
-  scoreElement = document.getElementById('score')
-  scoreCounter: number = 0
-  gameFlowHeight = 600
-  block = new Block()
-  counts: number = 0
-
-  checkTouch (item: any) {
-    const itemPosition = parseInt(window.getComputedStyle(item).getPropertyValue('top'))
-    const grMinHeight = 90
-    const grMaxHeight = 60
-    const goodMinHeight = 110
-    const goodMaxHeight = 30
-    const greatArea = itemPosition >= this.gameFlowHeight - grMinHeight && itemPosition <= this.gameFlowHeight - grMaxHeight
-    const goodArea = itemPosition >= this.gameFlowHeight - goodMinHeight && itemPosition <= this.gameFlowHeight - goodMaxHeight
-
-    if (greatArea) {
-      this.scoreCounter += 2
-    } else if (goodArea) {
-      this.scoreCounter += 1
+  isPlay: boolean = false
+  safeLoop: number = 0
+
+  get storeItems (): IArrowData[] {
```

Comment:
I'd give a more definite name. `items` can mean a lot
`arrowsData` says me 'I'm an object with data, rather than an array'.
My recommendation is to avoid names like `items`, `data`, `array`